webapp/src/main.rs

```diff
@@ -94,7 +93,30 @@ async fn main() -> Result<()> {
     let config =
         RustlsConfig::from_pem_file(cert_dir.join("cert.pem"), cert_dir.join("key.pem")).await?;
 
-    let app = router(Arc::new(rx), network);
+    let session_store = InMemorySessionStore::new();
+    let deletion_task = tokio::task::spawn(
+        session_store
+            .clone()
+            .continuously_delete_expired(Duration::from_secs(60)),
+    );
+
+    let session_layer = SessionManagerLayer::new(session_store.clone())
+        .with_secure(false)
```

Comment:
What is this and why shouldn't it be `true`? :D

Reply:
Thanks for pointing it out. This specifies whether the cookie should get the secure flag. We should definitely set this to true in production.